app/assets/stylesheets/style.css

```diff
@@ -450,3 +450,11 @@ pre {
   /* Hide the text; just using it for sizing */
   visibility: hidden;
 }
+
+.answer-accept{
+  font-weight: bold;
+}
+
+.answer-accept.btn-success{
```

Comment:
Sorry to nitpick; a space before the `{` here would be great; i'll fix in an extra commit this time. Thanks!